fieldservice/views/fsm_location.xml

```diff
@@ -22,7 +22,22 @@
         <field name="arch" type="xml">
             <form string="Location">
                 <sheet>
-                    <div class="oe_button_box" name="button_box"/>
+                    <div class="oe_button_box" position="inside">
+                        <button type="object"
+                                name="action_view_equipment"
+                                class="oe_stat_button"
+                                icon="fa-briefcase"
+                                attrs="{'invisible': [('equipment_count', '=', 0)]}" groups="base.group_user">
+                            <field name="equipment_count" widget="statinfo" string="Equipment"/>
+                        </button>                        
+                        <button type="object"
+                                name="action_view_contacts"
+                                class="oe_stat_button"
+                                icon="fa-user-circle-o"
+                                attrs="{'invisible': [('contact_count', '=', 0)]}" groups="base.group_user">
```

Comment:
Should be visible even if count is 0.

Reply:
In fieldservice_stock/views/fsm_order.xml line 22 and line 15 do not display their smart buttons when the count is 0. I will change mine, I just wanted to make sure you were aware in case that is an inconsistency.